fearless/Common/Model/MultiassetCryptoType.swift

```diff
@@ -19,4 +19,7 @@ extension MultiassetCryptoType {
             return .ecdsa
         }
     }
+
+    static let substrateTypes: [Self] = [.sr25519, .ed25519, .substrateEcdsa]
+    static let ethereumTypes: Self = .ethereumEcdsa
```

Comment:
I thought about it, but now there is no need for it. The advantage of a specific type instead of an array is that we don't need to expand the optional if we need to pass it somewhere

Reply:
I'll rename field to "ethereumType"